src/infra/main/template.yaml

```diff
@@ -144,10 +163,175 @@ Resources:
+      Timeout: 10
+      Environment:
+        Variables:
+          TABLE_NAME:
+            Fn::ImportValue: !Sub "${CoreStackName}-AccountStatusTableName"
+          METRIC_SERVICE_NAME: !Sub "${AWS::StackName}-StatusRetriever"
+      Handler: status-retriever-handler.handle
+      Role: !GetAtt StatusRetrieverFunctionRole.Arn
+      Tags:
+        CheckovRulesToSkip: CKV_AWS_116.CKV_AWS_115
+        Product: !Ref ProductTagValue
+        System: !Ref SystemTagValue
+        Environment: !Ref Environment
+        Owner: !Ref OwnerTagValue
+        Source: !Ref SourceTagValue
+
+  StatusRetrieverFunctionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName:  !Sub '${AWS::StackName}-StatusRetrieverFunctionRole'
+      PermissionsBoundary: !If [ UsePermissionsBoundary, !Ref PermissionsBoundary, !Ref AWS::NoValue ]
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: lambda.amazonaws.com
+            Action:
+              - "sts:AssumeRole"
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AWSXrayWriteOnlyAccess
+        - arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole
+      Tags:
+        - Key: Product
+          Value: !Ref ProductTagValue
+        - Key: System
+          Value: !Ref SystemTagValue
+        - Key: Environment
+          Value: !Ref Environment
+        - Key: Owner
+          Value: !Ref OwnerTagValue
+        - Key: Source
+          Value: !Ref SourceTagValue
+
+  StatusRetrieverFunctionPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: !Sub "${AWS::StackName}-StatusRetrieverFunctionPolicy"
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action:
+              - "dynamodb:Query"
+              - "dynamodb:GetItem"
+            Resource:
+              - Fn::ImportValue: !Sub "${CoreStackName}-AccountStatusTableArn"
+      Roles:
+        - !Ref StatusRetrieverFunctionRole
+
+  # Private API Gateway resources
+  PrivateRestApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      # checkov:skip=CKV_AWS_120: API Gateway caching not enabled
+      Name: !Sub "${AWS::StackName}-PrivateRestApi"
+      AlwaysDeploy: true
+      OpenApiVersion: 3.0.3
+      DefinitionBody:
+        Fn::Transform:
+          Name: AWS::Include
+          Parameters:
+            Location: "../../specs/api.yaml"
+      Description: "Private REST API that provides method to interact with the account interventions services"
+      StageName: !Ref ApiStageName
+      Auth:
+        ResourcePolicy:
+          IntrinsicVpcWhitelist:
+            - !FindInMap [ EnvConfig, !Ref Environment, AuthenticationVPCID ]
+            - !FindInMap [ EnvConfig, !Ref Environment, OneLoginHomeVPCID ]
+            - !FindInMap [ EnvConfig, !Ref Environment, OrchestrationVPCID ]
+      # AccessLogSetting:
```

Comment:
Commenting out the access log settings?

Reply:
and related log group

Reply:
Wasn't sure if this is needed or not, happy enough to add them in if needed?